Building `spice-gtk` fails because it cannot find Python's `pyparsing`

### Describe the bug
A regular update fails because `spice-gtk` apparently won't find the `pyparsing` module:

### Steps To Reproduce
*(edit: line-wrapped for clarity)*
```console-session
[root@chobotnice:~]# time nixos-rebuild switch --upgrade -j 1
unpacking channels...
building Nix...
building the system configuration...
these 17 derivations will be built:
  /nix/store/mgs5j3b4baqjw322k7gfhx8if3wgssq3-spice-gtk-0.37.drv
  /nix/store/gxk3rzv7vwsxajg0d0xd1yfdpwsmjpj1-virt-manager-3.2.0.drv
  /nix/store/b49spk35rxq2ysni0r5xfixizcq6iikg-system-path.drv
  /nix/store/3f55lr2rdahlpzf7blgn99lf0xqg1hk6-unit-systemd-fsck-.service.drv
  /nix/store/7jz3x4brmb7nn3qf0kvvjppcgdbx0g17-unit-accounts-daemon.service.drv
  /nix/store/l7lmqp75x9qmr2xmc45rk3407agqg5fj-unit-polkit.service.drv
  /nix/store/abi33c20bl1hq3y7lhb3kmcfmxbzrzva-dbus-1.drv
  /nix/store/lnfhl5fk6d9zxs6gjv3wg9bl1r2r9f9q-unit-dbus.service.drv
  /nix/store/6rsxw7m1y88iihq5gsfxhyr8wap5yb78-system-units.drv
  /nix/store/vqp09k6dfhx76pz5hyycxdbqsd273x0j-unit-dbus.service.drv
  /nix/store/ggwclaajcw499alhcfzy7fzrasq2ih8y-user-units.drv
  /nix/store/bd2f9xywpgk0qibfm30xgbw5pslvdlnq-etc.drv
  /nix/store/nn0m5l26sysbrki0kgyz58k569fxhagq-udev-rules.drv
  /nix/store/rmgb538c0sx95dv2wna72ns0mnmb67fl-link-units.drv
  /nix/store/qb7zgx327ng59wnqm4x4ksn6fm3200hg-stage-1-init.sh.drv
  /nix/store/bijnxsiqsmr3a10yhwpqrx0zkv9qy1yk-initrd-linux-5.16.7.drv
  /nix/store/s2gy6khwmgsmzd57lmfygs2lb668xgpk-nixos-system-chobotnice-22.05pre353535.48d63e924a2.drv
building '/nix/store/rmgb538c0sx95dv2wna72ns0mnmb67fl-link-units.drv'...
'/nix/store/24ljibki63lxk0m11qnw8fh9smh64g3x-systemd-249.7/lib/systemd/network/99-default.link' -> '/nix/store/si45fwgqrwzpkhzia0932ndm6q0br6pw-link-units/99-default.link'
building '/nix/store/mgs5j3b4baqjw322k7gfhx8if3wgssq3-spice-gtk-0.37.drv'...
unpacking sources
unpacking source archive /nix/store/1085xrxs0sl84r12641shac53af5cr18-spice-gtk-0.37.tar.bz2
source root is spice-gtk-0.37
setting SOURCE_DATE_EPOCH to timestamp 1558007590 of file spice-gtk-0.37/doc/reference/html/spice-gtk-spice-version.html
patching sources
configuring
meson flags: 
  --buildtype=plain
  --libdir=/nix/store/54ix55g2q6jrkvgda5290azz0rbj7jix-spice-gtk-0.37/lib
 --libexecdir=/nix/store/54ix55g2q6jrkvgda5290azz0rbj7jix-spice-gtk-0.37/libexec
         --bindir=/nix/store/54ix55g2q6jrkvgda5290azz0rbj7jix-spice-gtk-0.37/bin
 --sbindir=/nix/store/54ix55g2q6jrkvgda5290azz0rbj7jix-spice-gtk-0.37/sbin
         --includedir=/nix/store/10ncnws2amx9zfq8smq5g9dv8i2zd59h-spice-gtk-0.37-dev/include
         --mandir=/nix/store/wnhpgzxs3niq0qii41vbwgmvg9z1jicj-spice-gtk-0.37-man/share/man
 --infodir=/nix/store/54ix55g2q6jrkvgda5290azz0rbj7jix-spice-gtk-0.37/share/info
         --localedir=/nix/store/54ix55g2q6jrkvgda5290azz0rbj7jix-spice-gtk-0.37/share/locale
         -Dauto_features=enabled
         -Dwrap_mode=nodownload
         --prefix=/nix/store/54ix55g2q6jrkvgda5290azz0rbj7jix-spice-gtk-0.37
   -Dcelt051=disabled
   -Dpulse=disabled 
The Meson build system
Version: 0.60.3
Source dir: /build/spice-gtk-0.37
Build dir: /build/spice-gtk-0.37/build
Build type: native build
Project name: spice-gtk
Project version: 0.37
C compiler for the host machine: gcc (gcc 10.3.0 "gcc (GCC) 10.3.0")
C linker for the host machine: gcc ld.bfd 2.35.2
Host machine cpu family: x86_64
Host machine cpu: x86_64
Program sh found: YES (/nix/store/4nmqxajzaf60yjribkgvj5j54x9yvr1r-bash-5.1-p12/bin/sh)

Executing subproject spice-common 

spice-common| Project name: spice-common
spice-common| Project version: undefined
spice-common| C compiler for the host machine: gcc (gcc 10.3.0 "gcc (GCC) 10.3.0")
spice-common| C linker for the host machine: gcc ld.bfd 2.35.2
spice-common| Has header "alloca.h" : YES
spice-common| Has header "arpa/inet.h" : YES
spice-common| Has header "dlfcn.h" : YES
spice-common| Has header "inttypes.h" : YES
spice-common| Has header "malloc.h" : YES
spice-common| Has header "memory.h" : YES
spice-common| Has header "netinet/in.h" : YES
spice-common| Has header "stddef.h" : YES
spice-common| Has header "stdint.h" : YES
spice-common| Has header "stdlib.h" : YES
spice-common| Has header "strings.h" : YES
spice-common| Has header "string.h" : YES
spice-common| Has header "sys/socket.h" : YES
spice-common| Has header "sys/stat.h" : YES
spice-common| Has header "sys/types.h" : YES
spice-common| Has header "unistd.h" : YES
spice-common| Has header "regex.h" : YES
spice-common| Has header "sys/mman.h" : YES
spice-common| Checking for function "alloca" : YES
spice-common| Checking for function "sigaction" : YES
spice-common| Checking for function "drand48" : YES
spice-common| Checking for function "setlinebuf" : YES
spice-common| Checking for function "hypot" : NO
spice-common| Library m found: YES
spice-common| Checking for function "hypot" with dependency -lm: YES
spice-common| Found pkg-config: /nix/store/r255i42fsinsmjpywxzj8rsav51vx4b5-pkg-config-wrapper-0.29.2/bin/pkg-config (0.29.2)
spice-common| Run-time dependency spice-protocol found: YES 0.14.3
spice-common| Run-time dependency glib-2.0 found: YES 2.70.2
spice-common| Run-time dependency gio-2.0 found: YES 2.70.2
spice-common| Run-time dependency gthread-2.0 found: YES 2.70.2
spice-common| Run-time dependency pixman-1 found: YES 0.38.4
spice-common| Run-time dependency openssl found: YES 1.1.1m
spice-common| Dependency celt051 skipped: feature celt051 disabled
spice-common| Run-time dependency opus found: YES 1.3.1
spice-common| Program python3 found: YES (/nix/store/afi0ysqw20yiiw2gr2d28dx40bc4ddf8-python3-3.9.10/bin/python3.9)
spice-common| Message: Checking for python module six
spice-common| Message: Checking for python module pyparsing

subprojects/spice-common/meson.build:140:6: ERROR: Problem encountered: Python module pyparsing not found

A full log can be found at /build/spice-gtk-0.37/build/meson-logs/meson-log.txt
error: builder for '/nix/store/mgs5j3b4baqjw322k7gfhx8if3wgssq3-spice-gtk-0.37.drv' failed with exit code 1
error: 1 dependencies of derivation '/nix/store/gxk3rzv7vwsxajg0d0xd1yfdpwsmjpj1-virt-manager-3.2.0.drv' failed to build
error: 1 dependencies of derivation '/nix/store/b49spk35rxq2ysni0r5xfixizcq6iikg-system-path.drv' failed to build
building '/nix/store/nn0m5l26sysbrki0kgyz58k569fxhagq-udev-rules.drv'...
error: 1 dependencies of derivation '/nix/store/s2gy6khwmgsmzd57lmfygs2lb668xgpk-nixos-system-chobotnice-22.05pre353535.48d63e924a2.drv' failed to build

real    0m11.361s
user    0m9.417s
sys     0m2.046s
```
### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[root@chobotnice:~]# nix-shell -p nix-info --run "nix-info -m"
this path will be fetched (0.06 MiB download, 0.30 MiB unpacked):
  /nix/store/xsgq8c5pg6lpjils5qvg67crrv6sy247-bash-interactive-5.1-p12-dev
copying path '/nix/store/xsgq8c5pg6lpjils5qvg67crrv6sy247-bash-interactive-5.1-p12-dev' from 'https://cache.nixos.org'...
 - system: `"x86_64-linux"`
 - host os: `Linux 5.15.12, NixOS, 22.05 (Quokka)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.5.1`
 - channels(root): `"nixos-22.05pre353535.48d63e924a2"`
 - channels(jkt): `""`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

I just tried bumping spice-gtk to version 0.40 locally and it indeed passes that "check for dependencies" phase of the build. It fails later on though for reasons that I'm not familiar enough with the build process to understand.

With this patch:
```patch
diff --git a/pkgs/development/libraries/spice-gtk/default.nix b/pkgs/development/libraries/spice-gtk/default.nix
index 7df74ff1a5d..917c67a5cb2 100644
--- a/pkgs/development/libraries/spice-gtk/default.nix
+++ b/pkgs/development/libraries/spice-gtk/default.nix
@@ -11,6 +11,7 @@
 , gtk3
 , json-glib
 , libcacard
+, libcap_ng
 , libdrm
 , libjpeg_turbo
 , libopus
@@ -62,8 +63,8 @@ stdenv.mkDerivation rec {
   outputs = [ "out" "dev" "devdoc" "man" ];

   src = fetchurl {
-    url = "https://www.spice-space.org/download/gtk/${pname}-${version}.tar.bz2";
-    sha256 = "1drvj8y35gnxbnrxsipwi15yh0vs9ixzv4wslz6r3lra8w3bfa0z";
+    url = "https://www.spice-space.org/download/gtk/${pname}-${version}.tar.xz";
+    sha256 = "I/X/f6gLdWR85zzaXq+LMi80Mtu7f286g5Y0YYrbztM=";
   };

   postPatch = ''
@@ -98,6 +99,7 @@ stdenv.mkDerivation rec {
     gtk3
     json-glib
     libcacard
+    libcap_ng
     libdrm
     libjpeg_turbo
     libopus
@@ -113,11 +115,6 @@ stdenv.mkDerivation rec {

   PKG_CONFIG_POLKIT_GOBJECT_1_POLICYDIR = "${placeholder "out"}/share/polkit-1/actions";

-  mesonFlags = [
-    "-Dcelt051=disabled"
-    "-Dpulse=disabled" # is deprecated upstream
-  ];
-
   meta = with lib; {
     description = "GTK 3 SPICE widget";
     longDescription = ''
--
2.35.1
```

I get the following build log:

```console
$ nix log /nix/store/yhn026ws0wav4lz0yqv9z230q9wzsmlj-spice-gtk-0.40.drv | cat
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking sources
unpacking source archive /nix/store/0md7l2rdxpzqlm2jjjhzvpzs6bpkcpwj-spice-gtk-0.40.tar.xz
source root is spice-gtk-0.40
setting SOURCE_DATE_EPOCH to timestamp 1644485361 of file spice-gtk-0.40/THANKS
@nix { "action": "setPhase", "phase": "patchPhase" }
patching sources
@nix { "action": "setPhase", "phase": "configurePhase" }
configuring
meson flags: --buildtype=plain         --libdir=/nix/store/svjyjbhc5fibadjcimgxhjg40sghqih6-spice-gtk-0.40/lib --libexecdir=/nix/store/svjyjbhc5fibadjcimgxhjg40sghqih6-spice-gtk-0.40/libexec         --bindir=/nix/store/svjyjbhc5fibadjcimgxhjg40sghqih6-spice-gtk-0.40/bin --sbindir=/nix/store/svjyjbhc5fibadjcimgxhjg40sghqih6-spice-gtk-0.40/sbin         --includedir=/nix/store/dy7kh9zqn7npr78vdl9rrnbk8brj43c2-spice-gtk-0.40-dev/include         --mandir=/nix/store/xhbz7ln6kzv2ph0a7f42npkdwbxp5snv-spice-gtk-0.40-man/share/man --infodir=/nix/store/svjyjbhc5fibadjcimgxhjg40sghqih6-spice-gtk-0.40/share/info         --localedir=/nix/store/svjyjbhc5fibadjcimgxhjg40sghqih6-spice-gtk-0.40/share/locale         -Dauto_features=enabled         -Dwrap_mode=nodownload         --prefix=/nix/store/svjyjbhc5fibadjcimgxhjg40sghqih6-spice-gtk-0.40
The Meson build system
Version: 0.60.3
Source dir: /build/spice-gtk-0.40
Build dir: /build/spice-gtk-0.40/build
Build type: native build
Project name: spice-gtk
Project version: 0.40
C compiler for the host machine: gcc (gcc 10.3.0 "gcc (GCC) 10.3.0")
C linker for the host machine: gcc ld.bfd 2.35.2
Host machine cpu family: x86_64
Host machine cpu: x86_64
Program build-aux/meson-dist found: YES (/build/spice-gtk-0.40/build-aux/meson-dist)

Executing subproject spice-common

spice-common| Project name: spice-common
spice-common| Project version: undefined
spice-common| C compiler for the host machine: gcc (gcc 10.3.0 "gcc (GCC) 10.3.0")
spice-common| C linker for the host machine: gcc ld.bfd 2.35.2
spice-common| Has header "alloca.h" : YES
spice-common| Has header "arpa/inet.h" : YES
spice-common| Has header "dlfcn.h" : YES
spice-common| Has header "inttypes.h" : YES
spice-common| Has header "netinet/in.h" : YES
spice-common| Has header "stdlib.h" : YES
spice-common| Has header "sys/socket.h" : YES
spice-common| Has header "sys/stat.h" : YES
spice-common| Has header "sys/types.h" : YES
spice-common| Has header "unistd.h" : YES
spice-common| Has header "regex.h" : YES
spice-common| Has header "sys/mman.h" : YES
spice-common| Checking for function "alloca" : YES
spice-common| Checking for function "sigaction" : YES
spice-common| Checking for function "drand48" : YES
spice-common| Checking for function "setlinebuf" : YES
spice-common| Library m found: YES
spice-common| Found pkg-config: /nix/store/r255i42fsinsmjpywxzj8rsav51vx4b5-pkg-config-wrapper-0.29.2/bin/pkg-config (0.29.2)
spice-common| Run-time dependency spice-protocol found: YES 0.14.3
spice-common| Run-time dependency glib-2.0 found: YES 2.70.2
spice-common| Run-time dependency pixman-1 found: YES 0.38.4
spice-common| Run-time dependency openssl found: YES 1.1.1m
spice-common| Run-time dependency gio-2.0 found: YES 2.70.2
spice-common| Run-time dependency opus found: YES 1.3.1
spice-common| Program python3 found: YES (/nix/store/afi0ysqw20yiiw2gr2d28dx40bc4ddf8-python3-3.9.10/bin/python3)
spice-common| Message: Checking for python module six
spice-common| Message: Checking for python module pyparsing
spice-common| Run-time dependency libcacard found: YES 2.8.1
spice-common| Compiler for C supports arguments -DG_LOG_DOMAIN="Spice": YES
spice-common| Compiler for C supports arguments -Wno-unused-parameter: YES
spice-common| Compiler for C supports arguments -DGLIB_VERSION_MIN_REQUIRED=GLIB_VERSION_2_38: YES
spice-common| Compiler for C supports arguments -DGLIB_VERSION_MAX_ALLOWED=GLIB_VERSION_2_38: YES
spice-common| Run-time dependency gdk-pixbuf-2.0 found: YES 2.42.6
spice-common| Program asciidoc found: NO
spice-common| Configuring config.h using configuration
spice-common| Build targets in project: 14
spice-common| Subproject spice-common finished.


Executing subproject keycodemapdb

keycodemapdb| Project name: keycodemapdb
keycodemapdb| Project version: undefined
keycodemapdb| Build targets in project: 14
keycodemapdb| Subproject keycodemapdb finished.

Has header "termios.h" : YES
Has header "X11/XKBlib.h" : YES
Has header "sys/socket.h" : YES (cached)
Has header "sys/types.h" : YES (cached)
Has header "netinet/in.h" : YES (cached)
Has header "arpa/inet.h" : YES (cached)
Has header "valgrind/valgrind.h" : NO
Has header "epoxy/egl.h" : YES
Checking for function "clearenv" : YES
Checking for function "strtok_r" : YES
Dependency spice-protocol found: YES 0.14.3 (cached)
Dependency glib-2.0 found: YES 2.70.2 (cached)
Dependency gio-2.0 found: YES 2.70.2 (cached)
Run-time dependency gobject-2.0 found: YES 2.70.2
Dependency pixman-1 found: YES 0.38.4 (cached)
Dependency openssl found: YES 1.1.1m (cached)
Run-time dependency cairo found: YES 1.16.0
Run-time dependency libjpeg found: YES 2.1.2
Run-time dependency zlib found: YES 1.2.11
Run-time dependency json-glib-1.0 found: YES 1.6.6
Run-time dependency gio-unix-2.0 found: YES 2.70.2
Dependency opus found: YES 1.3.1 (cached)
Run-time dependency gtk+-3.0 found: YES 3.24.31
Run-time dependency epoxy found: YES 1.5.9
Run-time dependency x11 found: YES 1.7.2
Did not find CMake 'cmake'
Found CMake: NO
Run-time dependency libva-x11 found: NO (tried pkgconfig and cmake)
Has header "gtk-3.0/gdk/gdkwayland.h" : YES
Run-time dependency wayland-protocols found: YES 1.25
Found pkg-config: /nix/store/r255i42fsinsmjpywxzj8rsav51vx4b5-pkg-config-wrapper-0.29.2/bin/pkg-config (0.29.2)
Build-time dependency wayland-scanner found: YES 1.20.0
Program /nix/store/q2ps4p99ab5d3wldifrjwbm226i6jazp-wayland-1.20.0-bin/bin/wayland-scanner found: YES (/nix/store/q2ps4p99ab5d3wldifrjwbm226i6jazp-wayland-1.20.0-bin/bin/wayland-scanner)
Run-time dependency wayland-server found: YES 1.20.0
Run-time dependency wayland-cursor found: YES 1.20.0
Run-time dependency wayland-client found: YES 1.20.0
Run-time dependency libphodav-2.0 found: YES 2.5
Run-time dependency libsoup-2.4 found: YES 2.74.2
Run-time dependency gstreamer-1.0 found: YES 1.20.0
Run-time dependency gstreamer-base-1.0 found: YES 1.20.0
Run-time dependency gstreamer-app-1.0 found: YES 1.20.0
Run-time dependency gstreamer-audio-1.0 found: YES 1.20.0
Run-time dependency gstreamer-video-1.0 found: YES 1.20.0
Run-time dependency libusbredirparser-0.5 found: YES 0.12.0
Run-time dependency libusbredirhost found: YES 0.12.0
Run-time dependency libusb-1.0 found: YES 1.0.24
Run-time dependency libcap-ng found: YES 0.8.2
Run-time dependency polkit-gobject-1 found: YES 0.120
Checking for function "acl_get_file" : NO
Library acl found: YES
Checking for function "acl_get_file" with dependency -lacl: YES
Dependency gio-unix-2.0 found: YES 2.70.2 (cached)
Run-time dependency usbutils found: NO (tried pkgconfig and cmake)
Checking for function "makecontext" : YES
Checking for function "swapcontext" : YES
Checking for function "getcontext" : YES
Run-time dependency gobject-introspection-1.0 found: YES 1.70.0
Run-time dependency vapigen found: YES 0.54.6
Run-time dependency liblz4 found: YES 1.9.3
Run-time dependency libsasl2 found: YES 2.1.27
Dependency libcacard found: YES 2.8.1 (cached)
Compiler for C supports arguments -DHAVE_CONFIG_H: YES
Compiler for C supports arguments -DSPICE_COMPILATION: YES
Compiler for C supports arguments -DG_LOG_DOMAIN="GSpice": YES
Compiler for C supports arguments -Wno-sign-compare: YES
Compiler for C supports arguments -Wno-unused-parameter: YES (cached)
Compiler for C supports arguments -Wno-cast-function-type: YES
Compiler for C supports arguments -DGLIB_VERSION_MIN_REQUIRED=GLIB_VERSION_2_52: YES
Compiler for C supports arguments -DGLIB_VERSION_MAX_ALLOWED=GLIB_VERSION_2_52: YES
Compiler for C supports arguments -DGDK_VERSION_MIN_REQUIRED=GDK_VERSION_3_22: YES
Compiler for C supports arguments -DGDK_VERSION_MAX_ALLOWED=GDK_VERSION_3_22: YES
Configuring config.h using configuration
Configuring spice-version.h using configuration
Program glib-genmarshal found: YES (/nix/store/11fs0i9fmfd23vzdgsypfa1csx7rg9kx-glib-2.70.2-dev/bin/glib-genmarshal)
Program glib-mkenums found: YES (/nix/store/11fs0i9fmfd23vzdgsypfa1csx7rg9kx-glib-2.70.2-dev/bin/glib-mkenums)
Program glib-mkenums found: YES (/nix/store/11fs0i9fmfd23vzdgsypfa1csx7rg9kx-glib-2.70.2-dev/bin/glib-mkenums)
Compiler for C supports link arguments -Wl,--version-script=/build/spice-gtk-0.40/src/map-file: YES
Dependency gobject-introspection-1.0 found: YES 1.70.0 (cached)
Dependency gobject-introspection-1.0 found: YES 1.70.0 (cached)
Program g-ir-scanner found: YES (/nix/store/72rrr8m5jz1qpygc1ylkf6wd6611pllw-gobject-introspection-1.70.0-dev/bin/g-ir-scanner)
Dependency gobject-introspection-1.0 found: YES 1.70.0 (cached)
Program g-ir-compiler found: YES (/nix/store/72rrr8m5jz1qpygc1ylkf6wd6611pllw-gobject-introspection-1.70.0-dev/bin/g-ir-compiler)
Program ../build-aux/setcap-or-suid found: YES (/build/spice-gtk-0.40/src/../build-aux/setcap-or-suid)
Program glib-mkenums found: YES (/nix/store/11fs0i9fmfd23vzdgsypfa1csx7rg9kx-glib-2.70.2-dev/bin/glib-mkenums)
Program glib-mkenums found: YES (/nix/store/11fs0i9fmfd23vzdgsypfa1csx7rg9kx-glib-2.70.2-dev/bin/glib-mkenums)
Program python3 found: YES (/nix/store/afi0ysqw20yiiw2gr2d28dx40bc4ddf8-python3-3.9.10/bin/python3.9)
WARNING: extract_all_objects called without setting recursive
keyword argument. Meson currently defaults to
non-recursive to maintain backward compatibility but
the default will be changed in the future.

Program gtkdoc-scan found: YES (/nix/store/yhy4ri5a18fqv2mmcigcbkrbaiw4dij6-gtk-doc-1.33.2/bin/gtkdoc-scan)
Dependency glib-2.0 found: YES 2.70.2 (cached)
Dependency gtk+-3.0 found: YES 3.24.31 (cached)
Program gtkdoc-scan found: YES (/nix/store/yhy4ri5a18fqv2mmcigcbkrbaiw4dij6-gtk-doc-1.33.2/bin/gtkdoc-scan)
Program gtkdoc-scangobj found: YES (/nix/store/yhy4ri5a18fqv2mmcigcbkrbaiw4dij6-gtk-doc-1.33.2/bin/gtkdoc-scangobj)
Program gtkdoc-mkdb found: YES (/nix/store/yhy4ri5a18fqv2mmcigcbkrbaiw4dij6-gtk-doc-1.33.2/bin/gtkdoc-mkdb)
Program gtkdoc-mkhtml found: YES (/nix/store/yhy4ri5a18fqv2mmcigcbkrbaiw4dij6-gtk-doc-1.33.2/bin/gtkdoc-mkhtml)
Program gtkdoc-fixxref found: YES (/nix/store/yhy4ri5a18fqv2mmcigcbkrbaiw4dij6-gtk-doc-1.33.2/bin/gtkdoc-fixxref)
Program pod2man found: YES (/nix/store/4i0gfyhwf63ycz5j489mgkh2nlinnvfi-perl-5.34.0/bin/pod2man)
Program vapigen found: YES (/nix/store/2ibr4r9nqg18lnbvalr75pwjmv8qqjvc-vala-0.54.6/bin/vapigen)
Program vapigen found: YES (/nix/store/2ibr4r9nqg18lnbvalr75pwjmv8qqjvc-vala-0.54.6/bin/vapigen)
Build targets in project: 66

spice-gtk 0.40

    prefix                         : /nix/store/svjyjbhc5fibadjcimgxhjg40sghqih6-spice-gtk-0.40
    opus                           : YES
    gtk                            : YES
    wayland-protocols              : YES
    webdav                         : YES
    builtin-mjpeg                  : YES
    usbredir                       : YES
    libcap-ng                      : YES
    polkit                         : YES
    pie                            : YES
    usb-acl-helper-dir             : /nix/store/svjyjbhc5fibadjcimgxhjg40sghqih6-spice-gtk-0.40/libexec
    usb-ids-path                   :
    coroutine                      : ucontext
    introspection                  : YES
    vapi                           : YES
    lz4                            : YES
    sasl                           : YES
    smartcard                      : YES
    valgrind                       : NO
    libspice-client-glib.so version: 8.8.0
    libspice-client-gtk.so version : 5.1.0
    gtk_doc                        : YES

  Subprojects
    keycodemapdb                   : YES
    spice-common                   : YES

  User defined options
    auto_features                  : enabled
    bindir                         : /nix/store/svjyjbhc5fibadjcimgxhjg40sghqih6-spice-gtk-0.40/bin
    buildtype                      : plain
    includedir                     : /nix/store/dy7kh9zqn7npr78vdl9rrnbk8brj43c2-spice-gtk-0.40-dev/include
    infodir                        : /nix/store/svjyjbhc5fibadjcimgxhjg40sghqih6-spice-gtk-0.40/share/info
    libdir                         : /nix/store/svjyjbhc5fibadjcimgxhjg40sghqih6-spice-gtk-0.40/lib
    libexecdir                     : /nix/store/svjyjbhc5fibadjcimgxhjg40sghqih6-spice-gtk-0.40/libexec
    localedir                      : /nix/store/svjyjbhc5fibadjcimgxhjg40sghqih6-spice-gtk-0.40/share/locale
    mandir                         : /nix/store/xhbz7ln6kzv2ph0a7f42npkdwbxp5snv-spice-gtk-0.40-man/share/man
    prefix                         : /nix/store/svjyjbhc5fibadjcimgxhjg40sghqih6-spice-gtk-0.40
    sbindir                        : /nix/store/svjyjbhc5fibadjcimgxhjg40sghqih6-spice-gtk-0.40/sbin
    wrap_mode                      : nodownload

Found ninja-1.10.2 at /nix/store/84sc1p2fn4327x08k7qf1avghd55rbr4-ninja-1.10.2/bin/ninja
meson: enabled parallel building
@nix { "action": "setPhase", "phase": "buildPhase" }
building
build flags: -j8 -l8
[1/163] Compiling C object subprojects/spice-common/common/libspice-common.a.p/backtrace.c.o
[2/163] Compiling C object subprojects/spice-common/common/libspice-common.a.p/agent.c.o
[3/163] Compiling C object subprojects/spice-common/common/libspice-common.a.p/log.c.o
[4/163] Compiling C object subprojects/spice-common/common/libspice-common.a.p/canvas_utils.c.o
[5/163] Compiling C object subprojects/spice-common/common/libspice-common.a.p/mem.c.o
[6/163] Compiling C object subprojects/spice-common/common/libspice-common.a.p/marshaller.c.o
[7/163] Compiling C object subprojects/spice-common/common/libspice-common.a.p/snd_codec.c.o
[8/163] Compiling C object subprojects/spice-common/common/libspice-common.a.p/utils.c.o
[9/163] Compiling C object subprojects/spice-common/common/libspice-common.a.p/region.c.o
[10/163] Compiling C object subprojects/spice-common/tests/test_logging.p/test-logging.c.o
[11/163] Compiling C object subprojects/spice-common/tests/test_region.p/test-region.c.o
[12/163] Compiling C object subprojects/spice-common/tests/test_utils.p/test-utils.c.o
[13/163] Compiling C object subprojects/spice-common/common/libspice-common.a.p/lines.c.o
[14/163] Generating subprojects/spice-common/tests/test_marshallers with a custom command
Wrote subprojects/spice-common/tests/generated_test_marshallers.c
[15/163] Compiling C object subprojects/spice-common/common/libspice-common.a.p/pixman_utils.c.o
[16/163] Compiling C object subprojects/spice-common/tests/test_ssl_verify.p/test-ssl-verify.c.o
[17/163] Generating subprojects/spice-common/tests/test_marshallers_h with a custom command
Wrote subprojects/spice-common/tests/generated_test_marshallers.h
[18/163] Generating subprojects/spice-common/common/client_demarshallers with a custom command
Wrote subprojects/spice-common/common/generated_messages.h
Wrote subprojects/spice-common/common/generated_client_demarshallers.c
[19/163] Generating src/spice-marshal_c with a custom command
INFO: Reading ../src/spice-marshal.txt...
[20/163] Generating src/spice-marshal_h with a custom command
INFO: Reading ../src/spice-marshal.txt...
[21/163] Generating subprojects/spice-common/tests/test_demarshallers with a custom command
Wrote /build/spice-gtk-0.40/build/subprojects/spice-common/tests/generated_test_messages.h
Wrote subprojects/spice-common/tests/generated_test_demarshallers.c
[22/163] Generating subprojects/spice-common/tests/test_enums_h with a custom command
Wrote subprojects/spice-common/tests/generated_test_enums.h
[23/163] Compiling C object subprojects/spice-common/tests/test_quic.p/test-quic.c.o
[24/163] Compiling C object subprojects/spice-common/common/libspice-common.a.p/lz.c.o
[25/163] Compiling C object subprojects/spice-common/tests/test_marshallers.p/meson-generated_.._generated_test_demarshallers.c.o
[26/163] Generating src/spice-glib-enums.c with a custom command (wrapped by meson because command contains newlines, to capture output)
[27/163] Generating src/spice-glib-enums.h with a custom command (wrapped by meson because command contains newlines, to capture output)
[28/163] Compiling C object subprojects/spice-common/tests/test_marshallers.p/meson-generated_.._generated_test_marshallers.c.o
[29/163] Compiling C object subprojects/spice-common/tests/test_marshallers.p/test-marshallers.c.o
[30/163] Generating subprojects/spice-common/common/client_marshallers with a custom command
Wrote subprojects/spice-common/common/generated_client_marshallers.c
Wrote subprojects/spice-common/common/generated_client_marshallers.h
[31/163] Compiling C object src/spice-client-glib-usb-acl-helper.p/spice-client-glib-usb-acl-helper.c.o
[32/163] Generating src/spice-widget-enums.c with a custom command (wrapped by meson because command contains newlines, to capture output)
[33/163] Generating src/spice-widget-enums.h with a custom command (wrapped by meson because command contains newlines, to capture output)
[34/163] Generating src/vncdisplaykeymap_xorgevdev2xtkbd.h with a custom command (wrapped by meson to capture output)
[35/163] Generating src/vncdisplaykeymap_xorgkbd2xtkbd.h with a custom command (wrapped by meson to capture output)
[36/163] Compiling C object subprojects/spice-common/common/libspice-common-client.a.p/meson-generated_.._generated_client_marshallers.c.o
[37/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/meson-generated_.._spice-marshal.c.o
[38/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/meson-generated_.._spice-glib-enums.c.o
[39/163] Compiling C object subprojects/spice-common/common/libspice-common.a.p/quic.c.o
[40/163] Compiling C object subprojects/spice-common/common/libspice-common-client.a.p/ssl_verify.c.o
[41/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/channel-cursor.c.o
[42/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/channel-inputs.c.o
[43/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/channel-playback.c.o
[44/163] Compiling C object subprojects/spice-common/common/libspice-common-client.a.p/meson-generated_.._generated_client_demarshallers.c.o
[45/163] Linking static target subprojects/spice-common/common/libspice-common-client.a
[46/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/spice-session.c.o
[47/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/channel-port.c.o
[48/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/channel-display.c.o
[49/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/channel-record.c.o
[50/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/channel-smartcard.c.o
[51/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/channel-usbredir.c.o
[52/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/channel-main.c.o
[53/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/qmp-port.c.o
[54/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/smartcard-manager.c.o
[55/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/channel-webdav.c.o
[56/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/spice-client.c.o
[57/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/spice-audio.c.o
[58/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/spice-util.c.o
[59/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/spice-option.c.o
[60/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/channel-display-mjpeg.c.o
[61/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/bio-gio.c.o
[62/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/spice-gstaudio.c.o
[63/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/usb-device-manager.c.o
[64/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/channel-base.c.o
[65/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/decode-jpeg.c.o
[66/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/decode-zlib.c.o
[67/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/channel-display-gst.c.o
[68/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/spice-channel.c.o
[69/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/spice-glib-main.c.o
[70/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/gio-coroutine.c.o
[71/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/decode-glz.c.o
[72/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/spice-file-transfer-task.c.o
[73/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/continuation.c.o
[74/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/usb-acl-helper.c.o
[75/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/spice-uri.c.o
[76/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/giopipe.c.o
[77/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/vmcstream.c.o
[78/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/coroutine_ucontext.c.o
[79/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/client_sw_canvas.c.o
[80/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/usbutil.c.o
[81/163] Linking target src/spice-client-glib-usb-acl-helper
[82/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/usb-device-cd.c.o
[83/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/cd-usb-bulk-msd.c.o
[84/163] Generating src/vncdisplaykeymap_xorgxquartz2xtkbd.h with a custom command (wrapped by meson to capture output)
[85/163] Generating src/pointer-constraints-unstable-v1 client-header with a custom command
[86/163] Generating src/pointer-constraints-unstable-v1 server-header with a custom command
[87/163] Generating src/pointer-constraints-unstable-v1 private-code with a custom command
[88/163] Generating src/relative-pointer-unstable-v1 client-header with a custom command
[89/163] Generating src/relative-pointer-unstable-v1 server-header with a custom command
[90/163] Generating src/relative-pointer-unstable-v1 private-code with a custom command
[91/163] Generating src/vncdisplaykeymap_xorgxwin2xtkbd.h with a custom command (wrapped by meson to capture output)
[92/163] Generating src/vncdisplaykeymap_osx2xtkbd.h with a custom command (wrapped by meson to capture output)
[93/163] Generating src/vncdisplaykeymap_win322xtkbd.h with a custom command (wrapped by meson to capture output)
[94/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/usb-backend.c.o
[95/163] Compiling C object src/libspice-client-glib-2.0.so.8.8.0.p/cd-scsi.c.o
[96/163] Compiling C object tools/spicy-stats.p/spice-cmdline.c.o
[97/163] Compiling C object tools/spicy-stats.p/spicy-stats.c.o
[98/163] Linking static target tests/libtest-lib.a
[99/163] Compiling C object tools/spicy-screenshot.p/spicy-screenshot.c.o
[100/163] Compiling C object tools/spicy-screenshot.p/spice-cmdline.c.o
[101/163] Generating src/vncdisplaykeymap_x112xtkbd.h with a custom command (wrapped by meson to capture output)
[102/163] Compiling C object tools/spicy.p/spice-cmdline.c.o
[103/163] Compiling C object tests/test-util.p/util.c.o
[104/163] Compiling C object tests/test-uri.p/uri.c.o
[105/163] Compiling C object tools/spicy.p/spicy-connect.c.o
[106/163] Compiling C object tests/test-file-transfer.p/file-transfer.c.o
[107/163] Compiling C object src/libspice-client-gtk-3.0.so.5.1.0.p/spice-util.c.o
[108/163] Compiling C object src/libspice-client-gtk-3.0.so.5.1.0.p/spice-grabsequence.c.o
[109/163] Compiling C object src/libspice-client-gtk-3.0.so.5.1.0.p/meson-generated_.._pointer-constraints-unstable-v1-protocol.c.o
[110/163] Compiling C object src/libspice-client-gtk-3.0.so.5.1.0.p/meson-generated_.._relative-pointer-unstable-v1-protocol.c.o
[111/163] Compiling C object src/libspice-client-gtk-3.0.so.5.1.0.p/meson-generated_.._spice-marshal.c.o
[112/163] Compiling C object tools/spicy.p/spicy.c.o
[113/163] Compiling C object src/libspice-client-gtk-3.0.so.5.1.0.p/meson-generated_.._spice-widget-enums.c.o
[114/163] Compiling C object src/libspice-client-gtk-3.0.so.5.1.0.p/desktop-integration.c.o
[115/163] Compiling C object src/libspice-client-gtk-3.0.so.5.1.0.p/usb-device-widget.c.o
[116/163] Compiling C object src/libspice-client-gtk-3.0.so.5.1.0.p/vncdisplaykeymap.c.o
[117/163] Compiling C object src/libspice-client-gtk-3.0.so.5.1.0.p/spice-widget-cairo.c.o
[118/163] Compiling C object src/libspice-client-gtk-3.0.so.5.1.0.p/spice-gtk-session.c.o
[119/163] Compiling C object tests/test-coroutine.p/coroutine.c.o
[120/163] Generating po/it/LC_MESSAGES/spice-gtk-it.mo with a custom command
[121/163] Generating po/ru/LC_MESSAGES/spice-gtk-ru.mo with a custom command
[122/163] Compiling C object tests/test-session.p/session.c.o
[123/163] Compiling C object src/libspice-client-gtk-3.0.so.5.1.0.p/wayland-extensions.c.o
[124/163] Compiling C object tests/test-pipe.p/pipe.c.o
[125/163] Generating po/cs/LC_MESSAGES/spice-gtk-cs.mo with a custom command
[126/163] Generating po/de/LC_MESSAGES/spice-gtk-de.mo with a custom command
[127/163] Generating po/fr/LC_MESSAGES/spice-gtk-fr.mo with a custom command
[128/163] Generating man/spice-client.1 with a custom command
[129/163] Compiling C object tests/test-mock-acl-helper.p/mock-acl-helper.c.o
[130/163] Compiling C object tests/test-usb-acl-helper.p/usb-acl-helper.c.o
[131/163] Compiling C object src/libspice-client-gtk-3.0.so.5.1.0.p/spice-widget-egl.c.o
[132/163] Compiling C object src/libspice-client-gtk-3.0.so.5.1.0.p/spice-widget.c.o
[133/163] Compiling C object tests/test-cd-emu.p/cd-emu.c.o
[134/163] Compiling C object subprojects/spice-common/common/libspice-common.a.p/rop3.c.o
[135/163] Linking static target subprojects/spice-common/common/libspice-common.a
[136/163] Linking target subprojects/spice-common/tests/test_utils
[137/163] Linking target subprojects/spice-common/tests/test_marshallers
[138/163] Linking target subprojects/spice-common/tests/test_logging
[139/163] Linking target subprojects/spice-common/tests/test_region
[140/163] Linking target subprojects/spice-common/tests/test_quic
[141/163] Linking target subprojects/spice-common/tests/test_ssl_verify
[142/163] Linking target src/libspice-client-glib-2.0.so.8.8.0
[143/163] Generating symbol file src/libspice-client-glib-2.0.so.8.8.0.p/libspice-client-glib-2.0.so.8.8.0.symbols
[144/163] Linking target tests/test-util
[145/163] Linking target tests/test-uri
[146/163] Linking target tests/test-coroutine
[147/163] Linking target tests/test-pipe
[148/163] Linking target tools/spicy-stats
[149/163] Linking target tools/spicy-screenshot
[150/163] Linking target src/libspice-client-gtk-3.0.so.5.1.0
[151/163] Linking target tests/test-usb-acl-helper
[152/163] Linking target tests/test-mock-acl-helper
[153/163] Linking target tests/test-file-transfer
[154/163] Linking target tests/test-cd-emu
[155/163] Linking target tests/test-session
[156/163] Generating symbol file src/libspice-client-gtk-3.0.so.5.1.0.p/libspice-client-gtk-3.0.so.5.1.0.symbols
[157/163] Linking target tools/spicy
[158/163] Generating src/SpiceClientGLib-2.0.gir with a custom command
/build/spice-gtk-0.40/build/tmp-introspectnnx0uo17/SpiceClientGLib-2.0.c: In function ‘dump_object_type’:
/build/spice-gtk-0.40/build/tmp-introspectnnx0uo17/SpiceClientGLib-2.0.c:251:13: warning: Not available before 2.70
  251 |   if (G_TYPE_IS_FINAL (type))
      |             ^~~~~~~~~~~~~~~~~
/build/spice-gtk-0.40/build/tmp-introspectnnx0uo17/SpiceClientGLib-2.0.c: In function ‘dump_fundamental_type’:
/build/spice-gtk-0.40/build/tmp-introspectnnx0uo17/SpiceClientGLib-2.0.c:369:13: warning: Not available before 2.70
  369 |   if (G_TYPE_IS_FINAL (type))
      |             ^~~~~~~~~~~~~~~~~
g-ir-scanner: link: gcc -o /build/spice-gtk-0.40/build/tmp-introspectnnx0uo17/SpiceClientGLib-2.0 /build/spice-gtk-0.40/build/tmp-introspectnnx0uo17/SpiceClientGLib-2.0.o -L. -Wl,-rpath,. -Wl,--no-as-needed -L/build/spice-gtk-0.40/build/src -Wl,-rpath,/build/spice-gtk-0.40/build/src -L/build/spice-gtk-0.40/build/src -Wl,-rpath,/build/spice-gtk-0.40/build/src -L/nix/store/sxzai36pf2sk7dxwpk9hn1gqn94k4646-glib-2.70.2/lib -Wl,-rpath,/nix/store/sxzai36pf2sk7dxwpk9hn1gqn94k4646-glib-2.70.2/lib -L/nix/store/c901n9lvwila9q77zwnif901i4l3f8hc-pixman-0.38.4/lib -Wl,-rpath,/nix/store/c901n9lvwila9q77zwnif901i4l3f8hc-pixman-0.38.4/lib -L/nix/store/2aprvflf41z1dfma87mzd3md51xm2g71-openssl-1.1.1m/lib -Wl,-rpath,/nix/store/2aprvflf41z1dfma87mzd3md51xm2g71-openssl-1.1.1m/lib -L/nix/store/86rfybm6hlx40wk5xbzirx2yz7qz9q8p-libopus-1.3.1/lib -Wl,-rpath,/nix/store/86rfybm6hlx40wk5xbzirx2yz7qz9q8p-libopus-1.3.1/lib -L/nix/store/96fvdkyk4a2za2bq56gyg52gh30pfcnz-libcacard-2.8.1/lib -Wl,-rpath,/nix/store/96fvdkyk4a2za2bq56gyg52gh30pfcnz-libcacard-2.8.1/lib -L/nix/store/ybd7ni8vcg02k1w34l77c3axph8zx6yw-cairo-1.16.0/lib -Wl,-rpath,/nix/store/ybd7ni8vcg02k1w34l77c3axph8zx6yw-cairo-1.16.0/lib -L/nix/store/a6i6c7hkfn5s9bbgc1x6j1cnd5dppzkv-libjpeg-turbo-2.1.2/lib -Wl,-rpath,/nix/store/a6i6c7hkfn5s9bbgc1x6j1cnd5dppzkv-libjpeg-turbo-2.1.2/lib -L/nix/store/bqjh8dc6m5plswxrmmn9x4cpxrrw94lv-zlib-1.2.11/lib -Wl,-rpath,/nix/store/bqjh8dc6m5plswxrmmn9x4cpxrrw94lv-zlib-1.2.11/lib -L/nix/store/zi34wa1kxyz8jlf56bqnqk5slzk8q1h7-json-glib-1.6.6/lib -Wl,-rpath,/nix/store/zi34wa1kxyz8jlf56bqnqk5slzk8q1h7-json-glib-1.6.6/lib -L/nix/store/1a3f3rili87n35s5b4v5f5q1m9c1d5db-libsoup-2.74.2/lib -Wl,-rpath,/nix/store/1a3f3rili87n35s5b4v5f5q1m9c1d5db-libsoup-2.74.2/lib -L/nix/store/kr5jib1i6xmz8qm5gh5kj8rl74pr17ia-phodav-2.5-lib/lib -Wl,-rpath,/nix/store/kr5jib1i6xmz8qm5gh5kj8rl74pr17ia-phodav-2.5-lib/lib -L/nix/store/25ddjnb0blcq5bdkyzx5wvk2xq35zcc9-gstreamer-1.20.0/lib -Wl,-rpath,/nix/store/25ddjnb0blcq5bdkyzx5wvk2xq35zcc9-gstreamer-1.20.0/lib -L/nix/store/ab5xb6va2c5xxgr2sscygx56j4jc56aq-gst-plugins-base-1.20.0/lib -Wl,-rpath,/nix/store/ab5xb6va2c5xxgr2sscygx56j4jc56aq-gst-plugins-base-1.20.0/lib -L/nix/store/s4x09ks30zvd331bdg179ka7a7npfj6q-usbredir-0.12.0/lib -Wl,-rpath,/nix/store/s4x09ks30zvd331bdg179ka7a7npfj6q-usbredir-0.12.0/lib -L/nix/store/mxb07s27q830l54915snly4yiqvqkckk-libusb-1.0.24/lib -Wl,-rpath,/nix/store/mxb07s27q830l54915snly4yiqvqkckk-libusb-1.0.24/lib -L/nix/store/7abbl94528ka9xnc3z13vkzvrz4d9678-gobject-introspection-1.70.0/lib -Wl,-rpath,/nix/store/7abbl94528ka9xnc3z13vkzvrz4d9678-gobject-introspection-1.70.0/lib -L/nix/store/jvbg4dkpkkv3cn7hvb0nk4wgfk2jsb30-lz4-1.9.3/lib -Wl,-rpath,/nix/store/jvbg4dkpkkv3cn7hvb0nk4wgfk2jsb30-lz4-1.9.3/lib -L/nix/store/6rgjvy1a1qrx8yar257rnqlybah65078-cyrus-sasl-2.1.27/lib -Wl,-rpath,/nix/store/6rgjvy1a1qrx8yar257rnqlybah65078-cyrus-sasl-2.1.27/lib -lspice-client-glib-2.0 -lspice-client-glib-2.0 -lm -lglib-2.0 -lpixman-1 -lssl -lcrypto -lopus -lcacard -lgio-2.0 -lgobject-2.0 -lcairo -ljpeg -lz -ljson-glib-1.0 -lphodav-2.0 -lsoup-2.4 -lgmodule-2.0 -lgstreamer-1.0 -lgstbase-1.0 -lgstapp-1.0 -lgstaudio-1.0 -lgstvideo-1.0 -lusbredirparser -lusbredirhost -lusb-1.0 -lgirepository-1.0 -llz4 -lsasl2 -L/nix/store/sxzai36pf2sk7dxwpk9hn1gqn94k4646-glib-2.70.2/lib -lgio-2.0 -lgobject-2.0 -Wl,--export-dynamic -lgmodule-2.0 -pthread -lglib-2.0
[159/163] Generating src/SpiceClientGLib-2.0.typelib with a custom command
[160/163] Generating vapi/spice-client-glib-2.0.vapi with a custom command
SpiceClientGLib-2.0.gir:56.7-59.25: warning: Creation method of abstract class cannot be public.
[161/163] Generating src/SpiceClientGtk-3.0.gir with a custom command
/build/spice-gtk-0.40/build/tmp-introspectiotzd4u4/SpiceClientGtk-3.0.c: In function ‘dump_object_type’:
/build/spice-gtk-0.40/build/tmp-introspectiotzd4u4/SpiceClientGtk-3.0.c:251:13: warning: Not available before 2.70
  251 |   if (G_TYPE_IS_FINAL (type))
      |             ^~~~~~~~~~~~~~~~~
/build/spice-gtk-0.40/build/tmp-introspectiotzd4u4/SpiceClientGtk-3.0.c: In function ‘dump_fundamental_type’:
/build/spice-gtk-0.40/build/tmp-introspectiotzd4u4/SpiceClientGtk-3.0.c:369:13: warning: Not available before 2.70
  369 |   if (G_TYPE_IS_FINAL (type))
      |             ^~~~~~~~~~~~~~~~~
g-ir-scanner: link: gcc -o /build/spice-gtk-0.40/build/tmp-introspectiotzd4u4/SpiceClientGtk-3.0 /build/spice-gtk-0.40/build/tmp-introspectiotzd4u4/SpiceClientGtk-3.0.o -L. -Wl,-rpath,. -Wl,--no-as-needed -L/build/spice-gtk-0.40/build/src -Wl,-rpath,/build/spice-gtk-0.40/build/src -L/build/spice-gtk-0.40/build/src -Wl,-rpath,/build/spice-gtk-0.40/build/src -L/build/spice-gtk-0.40/build/src -Wl,-rpath,/build/spice-gtk-0.40/build/src -L/nix/store/sxzai36pf2sk7dxwpk9hn1gqn94k4646-glib-2.70.2/lib -Wl,-rpath,/nix/store/sxzai36pf2sk7dxwpk9hn1gqn94k4646-glib-2.70.2/lib -L/nix/store/c901n9lvwila9q77zwnif901i4l3f8hc-pixman-0.38.4/lib -Wl,-rpath,/nix/store/c901n9lvwila9q77zwnif901i4l3f8hc-pixman-0.38.4/lib -L/nix/store/2aprvflf41z1dfma87mzd3md51xm2g71-openssl-1.1.1m/lib -Wl,-rpath,/nix/store/2aprvflf41z1dfma87mzd3md51xm2g71-openssl-1.1.1m/lib -L/nix/store/86rfybm6hlx40wk5xbzirx2yz7qz9q8p-libopus-1.3.1/lib -Wl,-rpath,/nix/store/86rfybm6hlx40wk5xbzirx2yz7qz9q8p-libopus-1.3.1/lib -L/nix/store/96fvdkyk4a2za2bq56gyg52gh30pfcnz-libcacard-2.8.1/lib -Wl,-rpath,/nix/store/96fvdkyk4a2za2bq56gyg52gh30pfcnz-libcacard-2.8.1/lib -L/nix/store/ybd7ni8vcg02k1w34l77c3axph8zx6yw-cairo-1.16.0/lib -Wl,-rpath,/nix/store/ybd7ni8vcg02k1w34l77c3axph8zx6yw-cairo-1.16.0/lib -L/nix/store/a6i6c7hkfn5s9bbgc1x6j1cnd5dppzkv-libjpeg-turbo-2.1.2/lib -Wl,-rpath,/nix/store/a6i6c7hkfn5s9bbgc1x6j1cnd5dppzkv-libjpeg-turbo-2.1.2/lib -L/nix/store/bqjh8dc6m5plswxrmmn9x4cpxrrw94lv-zlib-1.2.11/lib -Wl,-rpath,/nix/store/bqjh8dc6m5plswxrmmn9x4cpxrrw94lv-zlib-1.2.11/lib -L/nix/store/zi34wa1kxyz8jlf56bqnqk5slzk8q1h7-json-glib-1.6.6/lib -Wl,-rpath,/nix/store/zi34wa1kxyz8jlf56bqnqk5slzk8q1h7-json-glib-1.6.6/lib -L/nix/store/1a3f3rili87n35s5b4v5f5q1m9c1d5db-libsoup-2.74.2/lib -Wl,-rpath,/nix/store/1a3f3rili87n35s5b4v5f5q1m9c1d5db-libsoup-2.74.2/lib -L/nix/store/kr5jib1i6xmz8qm5gh5kj8rl74pr17ia-phodav-2.5-lib/lib -Wl,-rpath,/nix/store/kr5jib1i6xmz8qm5gh5kj8rl74pr17ia-phodav-2.5-lib/lib -L/nix/store/25ddjnb0blcq5bdkyzx5wvk2xq35zcc9-gstreamer-1.20.0/lib -Wl,-rpath,/nix/store/25ddjnb0blcq5bdkyzx5wvk2xq35zcc9-gstreamer-1.20.0/lib -L/nix/store/ab5xb6va2c5xxgr2sscygx56j4jc56aq-gst-plugins-base-1.20.0/lib -Wl,-rpath,/nix/store/ab5xb6va2c5xxgr2sscygx56j4jc56aq-gst-plugins-base-1.20.0/lib -L/nix/store/s4x09ks30zvd331bdg179ka7a7npfj6q-usbredir-0.12.0/lib -Wl,-rpath,/nix/store/s4x09ks30zvd331bdg179ka7a7npfj6q-usbredir-0.12.0/lib -L/nix/store/mxb07s27q830l54915snly4yiqvqkckk-libusb-1.0.24/lib -Wl,-rpath,/nix/store/mxb07s27q830l54915snly4yiqvqkckk-libusb-1.0.24/lib -L/nix/store/7abbl94528ka9xnc3z13vkzvrz4d9678-gobject-introspection-1.70.0/lib -Wl,-rpath,/nix/store/7abbl94528ka9xnc3z13vkzvrz4d9678-gobject-introspection-1.70.0/lib -L/nix/store/jvbg4dkpkkv3cn7hvb0nk4wgfk2jsb30-lz4-1.9.3/lib -Wl,-rpath,/nix/store/jvbg4dkpkkv3cn7hvb0nk4wgfk2jsb30-lz4-1.9.3/lib -L/nix/store/6rgjvy1a1qrx8yar257rnqlybah65078-cyrus-sasl-2.1.27/lib -Wl,-rpath,/nix/store/6rgjvy1a1qrx8yar257rnqlybah65078-cyrus-sasl-2.1.27/lib -L/nix/store/38s9harni8shwjfl4m4x1z94niz6cqj7-gtk+3-3.24.31/lib -Wl,-rpath,/nix/store/38s9harni8shwjfl4m4x1z94niz6cqj7-gtk+3-3.24.31/lib -L/nix/store/7p8n0258bxpc2yq3f815vgivhaa0vl3w-atk-2.36.0/lib -Wl,-rpath,/nix/store/7p8n0258bxpc2yq3f815vgivhaa0vl3w-atk-2.36.0/lib -L/nix/store/2g610c5xqvikdixylipz01cymp1yb7j4-gdk-pixbuf-2.42.6/lib -Wl,-rpath,/nix/store/2g610c5xqvikdixylipz01cymp1yb7j4-gdk-pixbuf-2.42.6/lib -L/nix/store/s4agfw48asi7n81ajh12l6ghqr7fm9ph-pango-1.50.3/lib -Wl,-rpath,/nix/store/s4agfw48asi7n81ajh12l6ghqr7fm9ph-pango-1.50.3/lib -L/nix/store/nw2gv9l8lzi4scbddbg1vfvivzcpvag3-harfbuzz-3.2.0/lib -Wl,-rpath,/nix/store/nw2gv9l8lzi4scbddbg1vfvivzcpvag3-harfbuzz-3.2.0/lib -L/nix/store/jf6z41msq1kh4jg7m288fsfllrv7gca9-libepoxy-1.5.9/lib -Wl,-rpath,/nix/store/jf6z41msq1kh4jg7m288fsfllrv7gca9-libepoxy-1.5.9/lib -L/nix/store/6pdb0rz2w2hsfsy66kwrbz67jd424wv7-libX11-1.7.2/lib -Wl,-rpath,/nix/store/6pdb0rz2w2hsfsy66kwrbz67jd424wv7-libX11-1.7.2/lib -L/nix/store/pz2wx67gskw438ysws82nwk84whwd63d-wayland-1.20.0/lib -Wl,-rpath,/nix/store/pz2wx67gskw438ysws82nwk84whwd63d-wayland-1.20.0/lib -lspice-client-gtk-3.0 -lspice-client-glib-2.0 -lm -lglib-2.0 -lpixman-1 -lssl -lcrypto -lopus -lcacard -lgio-2.0 -lgobject-2.0 -lcairo -ljpeg -lz -ljson-glib-1.0 -lphodav-2.0 -lsoup-2.4 -lgmodule-2.0 -lgstreamer-1.0 -lgstbase-1.0 -lgstapp-1.0 -lgstaudio-1.0 -lgstvideo-1.0 -lusbredirparser -lusbredirhost -lusb-1.0 -lgirepository-1.0 -llz4 -lsasl2 -lgtk-3 -lgdk-3 -lpangocairo-1.0 -lpango-1.0 -lharfbuzz -latk-1.0 -lcairo-gobject -lgdk_pixbuf-2.0 -lepoxy -lX11 -lwayland-server -lwayland-cursor -lwayland-client -L/nix/store/sxzai36pf2sk7dxwpk9hn1gqn94k4646-glib-2.70.2/lib -lgio-2.0 -lgobject-2.0 -Wl,--export-dynamic -lgmodule-2.0 -pthread -lglib-2.0
[162/163] Generating src/SpiceClientGtk-3.0.typelib with a custom command
[163/163] Generating vapi/spice-client-gtk-3.0.vapi with a custom command
spice-client-glib-2.0.vapi:9.3-9.14: warning: Creation method of abstract class cannot be public.
@nix { "action": "setPhase", "phase": "glibPreInstallPhase" }
glibPreInstallPhase
@nix { "action": "setPhase", "phase": "glibPreInstallPhase" }
glibPreInstallPhase
@nix { "action": "setPhase", "phase": "installPhase" }
installing
install flags: install
[0/1] Installing files.
Installing src/spice-glib-enums.h to /nix/store/dy7kh9zqn7npr78vdl9rrnbk8brj43c2-spice-gtk-0.40-dev/include/spice-client-glib-2.0
Installing src/libspice-client-glib-2.0.so.8.8.0 to /nix/store/svjyjbhc5fibadjcimgxhjg40sghqih6-spice-gtk-0.40/lib
Installing src/SpiceClientGLib-2.0.gir to /nix/store/svjyjbhc5fibadjcimgxhjg40sghqih6-spice-gtk-0.40/share/gir-1.0
Installing src/SpiceClientGLib-2.0.typelib to /nix/store/svjyjbhc5fibadjcimgxhjg40sghqih6-spice-gtk-0.40/lib/girepository-1.0
Installing src/spice-client-glib-usb-acl-helper to /nix/store/svjyjbhc5fibadjcimgxhjg40sghqih6-spice-gtk-0.40/libexec
Installing src/spice-widget-enums.h to /nix/store/dy7kh9zqn7npr78vdl9rrnbk8brj43c2-spice-gtk-0.40-dev/include/spice-client-gtk-3.0
Installing src/libspice-client-gtk-3.0.so.5.1.0 to /nix/store/svjyjbhc5fibadjcimgxhjg40sghqih6-spice-gtk-0.40/lib
Installing src/SpiceClientGtk-3.0.gir to /nix/store/svjyjbhc5fibadjcimgxhjg40sghqih6-spice-gtk-0.40/share/gir-1.0
Installing src/SpiceClientGtk-3.0.typelib to /nix/store/svjyjbhc5fibadjcimgxhjg40sghqih6-spice-gtk-0.40/lib/girepository-1.0
Installing tools/spicy-stats to /nix/store/svjyjbhc5fibadjcimgxhjg40sghqih6-spice-gtk-0.40/bin
Installing tools/spicy-screenshot to /nix/store/svjyjbhc5fibadjcimgxhjg40sghqih6-spice-gtk-0.40/bin
Installing tools/spicy to /nix/store/svjyjbhc5fibadjcimgxhjg40sghqih6-spice-gtk-0.40/bin
Installing man/spice-client.1 to /nix/store/xhbz7ln6kzv2ph0a7f42npkdwbxp5snv-spice-gtk-0.40-man/share/man/man1
Installing po/cs/LC_MESSAGES/spice-gtk.mo to /nix/store/svjyjbhc5fibadjcimgxhjg40sghqih6-spice-gtk-0.40/share/locale/cs/LC_MESSAGES
Installing po/de/LC_MESSAGES/spice-gtk.mo to /nix/store/svjyjbhc5fibadjcimgxhjg40sghqih6-spice-gtk-0.40/share/locale/de/LC_MESSAGES
Installing po/fr/LC_MESSAGES/spice-gtk.mo to /nix/store/svjyjbhc5fibadjcimgxhjg40sghqih6-spice-gtk-0.40/share/locale/fr/LC_MESSAGES
Installing po/it/LC_MESSAGES/spice-gtk.mo to /nix/store/svjyjbhc5fibadjcimgxhjg40sghqih6-spice-gtk-0.40/share/locale/it/LC_MESSAGES
Installing po/ru/LC_MESSAGES/spice-gtk.mo to /nix/store/svjyjbhc5fibadjcimgxhjg40sghqih6-spice-gtk-0.40/share/locale/ru/LC_MESSAGES
Installing vapi/spice-client-glib-2.0.vapi to /nix/store/svjyjbhc5fibadjcimgxhjg40sghqih6-spice-gtk-0.40/share/vala/vapi
Installing vapi/spice-client-gtk-3.0.vapi to /nix/store/svjyjbhc5fibadjcimgxhjg40sghqih6-spice-gtk-0.40/share/vala/vapi
Installing /build/spice-gtk-0.40/build/src/spice-version.h to /nix/store/dy7kh9zqn7npr78vdl9rrnbk8brj43c2-spice-gtk-0.40-dev/include/spice-client-glib-2.0
Installing /build/spice-gtk-0.40/src/channel-cursor.h to /nix/store/dy7kh9zqn7npr78vdl9rrnbk8brj43c2-spice-gtk-0.40-dev/include/spice-client-glib-2.0
Installing /build/spice-gtk-0.40/src/channel-display.h to /nix/store/dy7kh9zqn7npr78vdl9rrnbk8brj43c2-spice-gtk-0.40-dev/include/spice-client-glib-2.0
Installing /build/spice-gtk-0.40/src/channel-inputs.h to /nix/store/dy7kh9zqn7npr78vdl9rrnbk8brj43c2-spice-gtk-0.40-dev/include/spice-client-glib-2.0
Installing /build/spice-gtk-0.40/src/channel-main.h to /nix/store/dy7kh9zqn7npr78vdl9rrnbk8brj43c2-spice-gtk-0.40-dev/include/spice-client-glib-2.0
Installing /build/spice-gtk-0.40/src/channel-playback.h to /nix/store/dy7kh9zqn7npr78vdl9rrnbk8brj43c2-spice-gtk-0.40-dev/include/spice-client-glib-2.0
Installing /build/spice-gtk-0.40/src/channel-port.h to /nix/store/dy7kh9zqn7npr78vdl9rrnbk8brj43c2-spice-gtk-0.40-dev/include/spice-client-glib-2.0
Installing /build/spice-gtk-0.40/src/channel-record.h to /nix/store/dy7kh9zqn7npr78vdl9rrnbk8brj43c2-spice-gtk-0.40-dev/include/spice-client-glib-2.0
Installing /build/spice-gtk-0.40/src/channel-smartcard.h to /nix/store/dy7kh9zqn7npr78vdl9rrnbk8brj43c2-spice-gtk-0.40-dev/include/spice-client-glib-2.0
Installing /build/spice-gtk-0.40/src/channel-usbredir.h to /nix/store/dy7kh9zqn7npr78vdl9rrnbk8brj43c2-spice-gtk-0.40-dev/include/spice-client-glib-2.0
Installing /build/spice-gtk-0.40/src/channel-webdav.h to /nix/store/dy7kh9zqn7npr78vdl9rrnbk8brj43c2-spice-gtk-0.40-dev/include/spice-client-glib-2.0
Installing /build/spice-gtk-0.40/src/qmp-port.h to /nix/store/dy7kh9zqn7npr78vdl9rrnbk8brj43c2-spice-gtk-0.40-dev/include/spice-client-glib-2.0
Installing /build/spice-gtk-0.40/src/smartcard-manager.h to /nix/store/dy7kh9zqn7npr78vdl9rrnbk8brj43c2-spice-gtk-0.40-dev/include/spice-client-glib-2.0
Installing /build/spice-gtk-0.40/src/spice-audio.h to /nix/store/dy7kh9zqn7npr78vdl9rrnbk8brj43c2-spice-gtk-0.40-dev/include/spice-client-glib-2.0
Installing /build/spice-gtk-0.40/src/spice-channel.h to /nix/store/dy7kh9zqn7npr78vdl9rrnbk8brj43c2-spice-gtk-0.40-dev/include/spice-client-glib-2.0
Installing /build/spice-gtk-0.40/src/spice-client.h to /nix/store/dy7kh9zqn7npr78vdl9rrnbk8brj43c2-spice-gtk-0.40-dev/include/spice-client-glib-2.0
Installing /build/spice-gtk-0.40/src/spice-file-transfer-task.h to /nix/store/dy7kh9zqn7npr78vdl9rrnbk8brj43c2-spice-gtk-0.40-dev/include/spice-client-glib-2.0
Installing /build/spice-gtk-0.40/src/spice-option.h to /nix/store/dy7kh9zqn7npr78vdl9rrnbk8brj43c2-spice-gtk-0.40-dev/include/spice-client-glib-2.0
Installing /build/spice-gtk-0.40/src/spice-session.h to /nix/store/dy7kh9zqn7npr78vdl9rrnbk8brj43c2-spice-gtk-0.40-dev/include/spice-client-glib-2.0
Installing /build/spice-gtk-0.40/src/spice-types.h to /nix/store/dy7kh9zqn7npr78vdl9rrnbk8brj43c2-spice-gtk-0.40-dev/include/spice-client-glib-2.0
Installing /build/spice-gtk-0.40/src/spice-uri.h to /nix/store/dy7kh9zqn7npr78vdl9rrnbk8brj43c2-spice-gtk-0.40-dev/include/spice-client-glib-2.0
Installing /build/spice-gtk-0.40/src/spice-util.h to /nix/store/dy7kh9zqn7npr78vdl9rrnbk8brj43c2-spice-gtk-0.40-dev/include/spice-client-glib-2.0
Installing /build/spice-gtk-0.40/src/usb-device-manager.h to /nix/store/dy7kh9zqn7npr78vdl9rrnbk8brj43c2-spice-gtk-0.40-dev/include/spice-client-glib-2.0
Installing /build/spice-gtk-0.40/src/spice-client-gtk.h to /nix/store/dy7kh9zqn7npr78vdl9rrnbk8brj43c2-spice-gtk-0.40-dev/include/spice-client-gtk-3.0
Installing /build/spice-gtk-0.40/src/spice-grabsequence.h to /nix/store/dy7kh9zqn7npr78vdl9rrnbk8brj43c2-spice-gtk-0.40-dev/include/spice-client-gtk-3.0
Installing /build/spice-gtk-0.40/src/spice-gtk-session.h to /nix/store/dy7kh9zqn7npr78vdl9rrnbk8brj43c2-spice-gtk-0.40-dev/include/spice-client-gtk-3.0
Installing /build/spice-gtk-0.40/src/spice-widget.h to /nix/store/dy7kh9zqn7npr78vdl9rrnbk8brj43c2-spice-gtk-0.40-dev/include/spice-client-gtk-3.0
Installing /build/spice-gtk-0.40/src/usb-device-widget.h to /nix/store/dy7kh9zqn7npr78vdl9rrnbk8brj43c2-spice-gtk-0.40-dev/include/spice-client-gtk-3.0
Installing /build/spice-gtk-0.40/data/org.spice-space.lowlevelusbaccess.policy to /nix/store/svjyjbhc5fibadjcimgxhjg40sghqih6-spice-gtk-0.40/share/polkit-1/actions
Installing /build/spice-gtk-0.40/build/meson-private/spice-client-glib-2.0.deps to /nix/store/svjyjbhc5fibadjcimgxhjg40sghqih6-spice-gtk-0.40/share/vala/vapi
Installing /build/spice-gtk-0.40/build/meson-private/spice-client-gtk-3.0.deps to /nix/store/svjyjbhc5fibadjcimgxhjg40sghqih6-spice-gtk-0.40/share/vala/vapi
Installing /build/spice-gtk-0.40/build/meson-private/spice-client-glib-2.0.pc to /nix/store/svjyjbhc5fibadjcimgxhjg40sghqih6-spice-gtk-0.40/lib/pkgconfig
Installing /build/spice-gtk-0.40/build/meson-private/spice-client-gtk-3.0.pc to /nix/store/svjyjbhc5fibadjcimgxhjg40sghqih6-spice-gtk-0.40/lib/pkgconfig
Running custom install script '/build/spice-gtk-0.40/src/../build-aux/setcap-or-suid /nix/store/svjyjbhc5fibadjcimgxhjg40sghqih6-spice-gtk-0.40/libexec/spice-client-glib-usb-acl-helper cap_fowner+ep'
--- stdout ---

--- stderr ---
/build/spice-gtk-0.40/src/../build-aux/setcap-or-suid: line 6: setcap: not found
Warning: setcap failed, falling back on suid!
chown: changing ownership of '/nix/store/svjyjbhc5fibadjcimgxhjg40sghqih6-spice-gtk-0.40/libexec/spice-client-glib-usb-acl-helper': Invalid argument
chmod: changing permissions of '/nix/store/svjyjbhc5fibadjcimgxhjg40sghqih6-spice-gtk-0.40/libexec/spice-client-glib-usb-acl-helper': Operation not permitted

FAILED: install script '/build/spice-gtk-0.40/src/../build-aux/setcap-or-suid /nix/store/svjyjbhc5fibadjcimgxhjg40sghqih6-spice-gtk-0.40/libexec/spice-client-glib-usb-acl-helper cap_fowner+ep' exit code 1, stopped
FAILED: meson-install
/nix/store/294w2dqblyqpn7rsx1hdk7sxyz4gmbsq-meson-0.60.3/bin/meson install --no-rebuild
ninja: build stopped: subcommand failed.
```

--- a/pkgs/development/libraries/spice-gtk/default.nix
+++ b/pkgs/development/libraries/spice-gtk/default.nix
@@ -9,8 +9,10 @@
 , gst_all_1
 , gtk-doc
 , gtk3
+, hwdata
 , json-glib
 , libcacard
+, libcap_ng
 , libdrm
 , libjpeg_turbo
 , libopus
@@ -28,8 +30,8 @@
 , python3
 , spice-protocol
 , usbredir
-, usbutils
 , vala
+, wayland-protocols
 , zlib
 , withPolkit ? true
 }:
@@ -57,19 +59,23 @@
 
 stdenv.mkDerivation rec {
   pname = "spice-gtk";
-  version = "0.37";
+  version = "0.40";
 
   outputs = [ "out" "dev" "devdoc" "man" ];
 
   src = fetchurl {
-    url = "https://www.spice-space.org/download/gtk/${pname}-${version}.tar.bz2";
-    sha256 = "1drvj8y35gnxbnrxsipwi15yh0vs9ixzv4wslz6r3lra8w3bfa0z";
+    url = "https://www.spice-space.org/download/gtk/${pname}-${version}.tar.xz";
+    sha256 = "sha256-I/X/f6gLdWR85zzaXq+LMi80Mtu7f286g5Y0YYrbztM=";
   };
 
   postPatch = ''
     # get rid of absolute path to helper in store so we can use a setuid wrapper
     substituteInPlace src/usb-acl-helper.c \
       --replace 'ACL_HELPER_PATH"/' '"'
+    # don't try to setcap/suid in a nix builder
+    substituteInPlace src/meson.build \
+      --replace "meson.add_install_script('../build-aux/setcap-or-suid'," \
+      "# meson.add_install_script('../build-aux/setcap-or-suid',"
   '';
 
   nativeBuildInputs = [
@@ -98,6 +104,7 @@ stdenv.mkDerivation rec {
     gtk3
     json-glib
     libcacard
+    libcap_ng
     libdrm
     libjpeg_turbo
     libopus
@@ -108,14 +115,15 @@ stdenv.mkDerivation rec {
     pixman
     spice-protocol
     usbredir
+    wayland-protocols
     zlib
-  ] ++ lib.optionals withPolkit [ polkit acl usbutils ] ;
+  ] ++ lib.optionals withPolkit [ polkit acl ] ;
 
   PKG_CONFIG_POLKIT_GOBJECT_1_POLICYDIR = "${placeholder "out"}/share/polkit-1/actions";
 
   mesonFlags = [
-    "-Dcelt051=disabled"
-    "-Dpulse=disabled" # is deprecated upstream
+    "-Dusb-acl-helper-dir=${placeholder "out"}/bin"
+    "-Dusb-ids-path=${hwdata}/share/hwdata/usb.ids"
   ];
 
   meta = with lib; {

